Michelf/Markdown.inc.php

```diff
@@ -1,10 +1,16 @@
 <?php
-
-# Use this file if you cannot use class autoloading. It will include all the
-# files needed for the Markdown parser.
-#
-# Take a look at the PSR-0-compatible class autoloading implementation
-# in the Readme.php file if you want a simple autoloader setup.
+/**
+ * Use this file if you cannot use class autoloading. It will include all the
+ * files needed for the Markdown parser.
+ *
+ * Take a look at the PSR-0-compatible class autoloading implementation
+ * in the Readme.php file if you want a simple autoloader setup.
+ *
+ * @package   php-markdown
+ * @author    Michael Fortin <michael.fortin@michaelf.com>
```

Comment:
That should actually be `Michel Fortin <michel.fortin@michelf.com>`. There's no "a" in my name.